drivers/video/video_mcux_csi.c

```diff
@@ -32,25 +32,6 @@ struct video_mcux_csi_data {
 	struct k_poll_signal *signal;
 };
 
-static inline unsigned int video_pix_fmt_bpp(uint32_t pixelformat)
```

Comment:
It seems like the `stm32_dcmi` driver also has a definition like this- should we clean that up here too?